stackslib/src/net/chat.rs

```diff
@@ -1969,9 +1956,29 @@ impl ConversationP2P {
         ) {
             Ok(Some(chunk)) => chunk,
             Ok(None) => {
-                // request for a stale chunk
+                // TODO: this is racey
```

Comment:
Same comment about the todo. Can you open an issue for this and link it here?

Reply:
https://github.com/stacks-network/stacks-core/issues/5067